### DISCUSSION TITLE
Names of the zones differ from what obtained from command line?

### DISCUSSION THREAD

**Discussion by e1ns0f** [cockpit-project/cockpit#19483] [Q&A]
Hello everyone! 

This is my first post and I am new to cockpit (which I am using with Fedora Server 38 on a RaspberryPi). I have realised that cockpit, at least for me, is showing names of firewall zones that are different compared to those showed by command line (I got Public in cockpit while cli is giving back FedoraServer). It is similar to the behaviour described here https://bugzilla.redhat.com/show_bug.cgi?id=1804919 where however is reported as fixed. Is this something that should be reported (I am very new to open source community) or it is a problem only of my system (my cockpit version is 301)? 

Thank you very much for your help in understanding and the suggestions on how to proceed.

**Comment by martinpitt** [comment]
Indeed https://bugzilla.redhat.com/show_bug.cgi?id=1804919 was the opposite direction. What exactly do you run when you say "run cli"?  The default zone name **is** "Public", that's also what firewalld CLI says at least for a Fedora 38 cloud image:
```
# firewall-cmd --get-active-zones 
public
  interfaces: eth0
```

There also is a "FedoraServer" zone, but it's not active by default:
```
# firewall-cmd --list-all-zones|grep -i ^[a-z]
FedoraServer
FedoraWorkstation
block
dmz
drop
external
home
internal
libvirt
libvirt-routed
nm-shared
public (active)
trusted
work
```

How is that looking for you?

**Reply by e1ns0f** [reply]
Thanks for the reply! This is what I get

```
# firewall-cmd --get-active-zones  
FedoraServer  
 interfaces: enu1u1u1
```

While in cockpit, if I click on the link "1 active zone" behind the Firewall switch in the networking tab I see :

![Screenshot_20231016_123104_Chrome](https://github.com/cockpit-project/cockpit/assets/11735200/9ce320d1-e717-4dad-9609-431a5e7d0ac9)

This is what was confusing me, I hope I am not messing up. This is also a fresh installation of Fedora Server 38 (ARM) on a raspberry pi.
